### PR TITLE
Breakfast Preparation Pull Request

### PR DESCRIPTION
**Pull request by Tiffany-Shumate**
Added equipment for breakfast!

## 🎯 Objective

Add the time to prepare breakfast

_Note why you're making this change and any relevant issues_

Currently, no time added in the recipe

## 🛠️ Changes made

_Note what you actual changes you're proposing_ 

add 30 minutes for time to prepare

## 👥 Who needs to review this Pull Request? 

_@mention some teammates to review your work._
_Pull Requests should include commits from each member of your group._

@jklee-bear @spg222

### CODE REVIEW

**Comment by Tiffany-Shumate** [comment]
Adding time to prepare breakfast @spg222 @myronhc

File: breakfast/readme.md [hunk 1]
```diff
@@ -7,7 +7,7 @@
 ## Ingredients:
```
Comment (Tiffany-Shumate) [inline breakfast/readme.md:7]: We will need eggs, flour

Comment (Tiffany-Shumate) [inline breakfast/readme.md:7]: We will need eggs

**Review by spg222 — APPROVED** [review]

File: breakfast/readme.md [hunk 2]
```diff
@@ -7,7 +7,8 @@
...
 
 
-## Equipment needed: We will need serving utensils, a pot, and utensils. 
+## Equipment needed 
+- We will need serving utensils, cutlery, spatula, and a frying pan!
```
Comment (Tiffany-Shumate) [inline breakfast/readme.md:12]: We will need serving plates for 4

Comment (Tiffany-Shumate) [inline breakfast/readme.md:12]: We will need serving plates for 4

Comment (jklee-bear) [reply breakfast/readme.md:12]: Looks good to me!

**Comment by jklee-bear** [comment]
It would be best if we have a waffle machine but I think we can try with frying pan!

**Pull request merged**